dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -58,12 +58,12 @@ BEGIN
     LTEXT "<Message>", IDC_SECURITY_MESSAGE, 15, 62, 245, 18
     LTEXT "<Logon date>", IDC_SECURITY_LOGONDATE, 15, 83, 245, 8
     CTEXT "Ce acțiune preferați?", IDC_STATIC, 10, 108, 255, 8
-    PUSHBUTTON "&Blochează calculatorul", IDC_SECURITY_LOCK, 7, 124, 80, 14
-    PUSHBUTTON "&Deautentifică-mă…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
-    PUSHBUTTON "Închid&e calculatorul…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
-    PUSHBUTTON "Schim&bă parola…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
-    PUSHBUTTON "&Gestionar de activități", IDC_SECURITY_TASKMGR, 98, 143, 80, 14
-    PUSHBUTTON "A&nulează", IDCANCEL, 188, 143, 80, 14
+    PUSHBUTTON "&Blocare calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
+    PUSHBUTTON "&Deautentificare…", IDC_SECURITY_LOGOFF, 98, 124, 80, 14
+    PUSHBUTTON "Închid&erer…", IDC_SECURITY_SHUTDOWN, 188, 124, 80, 14
+    PUSHBUTTON "Schim&bare parolă…", IDC_SECURITY_CHANGEPWD, 7, 143, 80, 14
```

Comment:
This line introduces an accelerator collision with the line
 PUSHBUTTON "&Blocare calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14

Reply:
No you didn't fix that properly! Now 
 PUSHBUTTON "Bloca&re calculator", IDC_SECURITY_LOCK, 7, 124, 80, 14
collides with
PUSHBUTTON "&Revocare", IDCANCEL, 188, 143, 80, 14

Reply:
I fixed it. Look in the last commit.

Reply:
I do confirm that The collisions are gone now. I do still insist on a screenshot. Download a Romanian 2k3 if you don't have it on your current more modern Windows.